www.foia.gov/_sass/modules/_sidebar_timeline.scss

```diff
@@ -0,0 +1,31 @@
+.sidebar_progress-bar {
+  margin: 0 0 0 0.8rem; // to accommodate left half of bubbles
```

Comment:
Do we want to use pixels here instead of `rems`? It looks slightly off-center in my browser.
![screenshot from 2017-09-27 14-10-29](https://user-images.githubusercontent.com/509703/30938106-cc4a5e0c-a38d-11e7-82d1-3757f8dea42e.png)